Tests/Tests/ConfigurationServiceSpec.swift

```diff
@@ -1,11 +1,13 @@
 import Foundation
 import Quick
 import Nimble
-#if canImport(RSDKUtilsTestHelpers)
-import class RSDKUtilsTestHelpers.URLSessionMock // SPM version
-#else
+
+#if canImport(RSDKUtils)
 import class RSDKUtils.URLSessionMock
+#else // SPM version
+import class RSDKUtilsTestHelpers.URLSessionMock
 #endif
```

Comment:
@imaginaris I think that we should use `#if SWIFT_PACKAGE` here?

Reply:
@malayli `SWIFT_PACKAGE` macro doesn't work in app or test source code. It can be only used inside the SDK code

Reply:
Sorry it looks like I was wrong. I don't know what was the problem last time. I added additional import fixes

Reply:
It seems like `SWIFT_PACKAGE` is false for UITests targets in SampleSPM project 🤷

Reply:
I see.